python/sdk/merlin/model.py

```diff
@@ -1137,11 +1141,19 @@ def create_prediction_job(self, job_config: PredictionJobConfig, sync: bool = Tr
                                                                                    job_id=j.id)
                 return PredictionJob(j, self._api_client)
             else:
-                j = job_client.models_model_id_versions_version_id_jobs_job_id_get(model_id=self.model.id,
-                                                                                   version_id=self.id,
-                                                                                   job_id=j.id)
+                try:
+                    j = job_client.models_model_id_versions_version_id_jobs_job_id_get(model_id=self.model.id,
+                                                                                       version_id=self.id,
+                                                                                       job_id=j.id)
+                    retry = DEFAULT_API_CALL_RETRY
+                except Exception:
+                        retry -= 1
+                        if retry == 0:
+                            j.status = "failed"
+                            break
+                        sleep(DEFAULT_PREDICTION_JOB_DELAY)
```

Comment:
It should be `DEFAULT_PREDICTION_JOB_RETRY_DELAY`

Reply:
Fixed, thanks!